nebula/ui/components/MZList.qml

```diff
@@ -3,19 +3,117 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import QtQuick 2.5
+import QtQuick.Controls
+
+import Mozilla.Shared 1.0
+
+// ListView with scroll bar and an ensureVisible() method that scrolls an item into the
+// ListView's visible viewport. (ensureVisible() is used by scrollToComponent())
 
 ListView {
     id: list
 
-    property var listName
+    property var _accessibleName: ""
+
     height:contentHeight
     Accessible.role: Accessible.List
-    Accessible.name: listName
+    Accessible.name: _accessibleName
     Accessible.ignored: !visible
-    activeFocusOnTab: true
-    interactive: false // disable scrolling on list since the entire window is scrollable
     boundsBehavior: Flickable.StopAtBounds
     highlightFollowsCurrentItem: true
     Keys.onDownPressed: list.incrementCurrentIndex()
     Keys.onUpPressed: list.decrementCurrentIndex()
+
+    ScrollBar.vertical: ScrollBar {
```

Comment:
A nice follow-up would be to make this a component for easier keeping in sync with MZFlickable